addons/mail/static/src/discuss/core/common/store_service_patch.js

```diff
@@ -36,6 +36,17 @@ const storeServicePatch = {
         channel.open();
         return channel;
     },
+    async fetchChannel(channelId) {
+        const channelIds = this.fetchParams.find(
+            (fetchParams) => fetchParams[0] === "discuss.channel"
+        );
+        if (channelIds) {
+            channelIds[1].push(channelId);
+            await this.fetchDeferred;
+        } else {
+            await this.fetchStoreData("discuss.channel", [channelId]);
+        }
```

Comment:
Hard to tell what's shape of fetchParams with type and naming destructured param.

Would be nice to have at the very least a jsdoc on `fetchParams` to understand the params shape.
I see that this could be a string, which is ok with `[0]` being 1st char, but `[1]` could theoretically crash, and also this seems like accidental "no problem".

Reply:
It's indeed only str or tuple(array) <str, any>.